tests/test_estimation/test_apply_imputation_link.py

```diff
@@ -65,5 +67,9 @@ def test_mc_imputation_types(self, mc_fimc_test_data):
             imputation_types=("c", "mc", "fir", "bir", "fimc", "fic"),
         )
         actual_output.drop(columns=["man_link"], inplace=True)
+        expected_output["target"] = expected_output[["target", "imputed_value"]].agg(
```

Comment:
This should not be here?

Reply:
target and imputation_value need to be separate in input data for unit test. Otherwise the code cannot tell if its an actual return or expected return from the function. I found it easier to combine the expected after creating input data for the function. This is only processing expected output and not function ouputs